src/ts/commands.ts

```diff
@@ -322,26 +321,35 @@ const maybeApplySuggestions = (
     return false;
   }
 
-  if (dispatch) {
-    const tr = state.tr;
-    suggestionsToApply.forEach(
-      ({ from, to, text }) => {
-        if (!text) {
-          return;
+  if (!dispatch) {
+    return true;
+  }
+
+  const tr = state.tr;
+  suggestionsToApply.forEach(({ from, to, text }) => {
+    if (!text) {
+      return;
+    }
+    const mappedFrom = tr.mapping.map(from);
+    const mappedTo = tr.mapping.map(to);
```

Comment:
`nit` Should these consts live inside the `getReplacementFragmentsFromReplacement` function?

Reply:
Great spot – I was unsure whether this was necessary when first implementing!